web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -84,6 +85,7 @@
     private String appName;
     private static final String AGENT_LOCATOR_INTERFACE_NAME = "com.microsoft.applicationinsights."
         + "agent.internal.coresync.AgentNotificationsHandler";
+    private String filterName = FILTER_NAME;
```

Comment:
Where is this field used for? Your commit message says - added fitlerName field "just in case someone tries to add tow instances"

Reply:
line 289 and 295. set on line 180

Reply:
Thanks makes sense. Just a question will this field ever be null? For example if user forgets to specify filterName?

Reply:
I don't think it'll be null, possibly empty or whitespace (it's up to the J2EE spec), but I can add a wrapper when it's used to guard against that.

Reply:
Actually, String.format handles nulls, so it's fine.